src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -333,3 +333,91 @@ ov::pass::GeluFusionWithTanhNoPower::GeluFusionWithTanhNoPower() {
     auto m = std::make_shared<pattern::Matcher>(mul6, matcher_name);
     this->register_matcher(m, callback);
 }
+
+ov::pass::GeluFusionWithTanhNoPower2::GeluFusionWithTanhNoPower2() {
+    MATCHER_SCOPE(GeluFusionWithTanhNoPower2);
+    // Replaces a sub-graph with a Gelu (ov::op::v0::Tanh) op
+    // Gaussian Error Linear Unit, TanH based approximation:
+    // x * (0.5 * (1 + tanh([sqrt(2 / pi)] * [x + 0.044715 * x * x * x])))
+
+    auto input = pass::pattern::any_input();
+
+    auto mul_0 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({input, input});
+    auto mul_1 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({input, mul_0});
+
+    auto mul_2_constant = ov::pass::pattern::wrap_type<ov::op::v0::Constant>();
+    auto mul_2 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({mul_1, mul_2_constant});
+
+    auto add_0 = ov::pass::pattern::wrap_type<ov::op::v1::Add>({input, mul_2});
+
+    auto mul_3_constant = ov::pass::pattern::wrap_type<ov::op::v0::Constant>();
+    auto mul_3 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({add_0, mul_3_constant});
+
+    auto tanh = ov::pass::pattern::wrap_type<ov::op::v0::Tanh>({mul_3});
+
+    auto add_1_constant = ov::pass::pattern::wrap_type<ov::op::v0::Constant>();
+    auto add_1 = ov::pass::pattern::wrap_type<ov::op::v1::Add>({tanh, add_1_constant});
+
+    auto mul_4_constant = ov::pass::pattern::wrap_type<ov::op::v0::Constant>();
+
+    // x * (0.5 * (1 + tanh))
+    auto mul_4_1 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({add_1, mul_4_constant});
+    auto mul_5_1 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({input, mul_4_1});
+
+    // (x * 0.5) * (1 + tanh)
+    auto mul_4_2 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({input, mul_4_constant});
+    auto mul_5_2 = ov::pass::pattern::wrap_type<ov::op::v1::Multiply>({add_1, mul_4_2});
+
+    // (0.5 * x) * (1 + tanh)
```

Comment:
This case will be matched automatically with `// (x * 0.5) * (1 + tanh)` if we move constant validations to predicate.
please double check

Reply:
There are some mistake in the description. probably we need it.